### DISCUSSION TITLE
Update comfyui always crash my python.exe - Entry Point not Found

### DISCUSSION THREAD

**Discussion by garland92** [Comfy-Org/ComfyUI#4498] [Q&A] [answered]
Update always uninstall my latest torch :
  
  Attempting uninstall: torch
    Found existing installation: torch 2.4.0+cu124
    Uninstalling torch-2.4.0+cu124:
      Successfully uninstalled torch-2.4.0+cu124
  Attempting uninstall: aiohttp
    Found existing installation: aiohttp 3.10.4
    Uninstalling aiohttp-3.10.4:
      Successfully uninstalled aiohttp-3.10.4
  Attempting uninstall: torchvision
    Found existing installation: torchvision 0.19.0+cu124
    Uninstalling torchvision-0.19.0+cu124:
      Successfully uninstalled torchvision-0.19.0+cu124
  Attempting uninstall: torchaudio
    Found existing installation: torchaudio 2.4.0+cu124
    Uninstalling torchaudio-2.4.0+cu124:
      Successfully uninstalled torchaudio-2.4.0+cu124
      

can you please don't make force uninstalled torch, torchvision, torchaudio or something. Everytime update it make my torch become torch-2.1.2+cu121.dist-info.

**Comment by ltdrdata** [comment]
Is this a phenomenon that occurs when updating the portable version using an update .bat file?

**Reply by garland92** [reply]
> Is this a phenomenon that occurs when updating the portable version using an update .bat file?

I'm update comfyui from stability matrix

**Reply by ltdrdata** [reply, marked as answer]
This is an issue that should be requested to StabilityMatrix.

**Reply by garland92** [reply]
> This is an issue that should be requested to StabilityMatrix.

My bad, Thanks for tell me.
